shoryuken.gemspec

```
@@ -24,4 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'aws-sdk-sqs', '>= 1.66.0'
   spec.add_dependency 'concurrent-ruby'
   spec.add_dependency 'thor'
+
+  spec.required_ruby_version = '>= 3.0.0'
```

RuboCop (1.69.1)

[warning] `required_ruby_version` and `TargetRubyVersion` (2.4, which may be specified in .rubocop.yml) should be equal.

(Gemspec/RequiredRubyVersion)